deno.json

```diff
@@ -28,7 +28,10 @@
     "singleQuote": true
   },
   "lint": {
-    "exclude": ["node_modules/", "vendor/", "dist/", "build/", "test/assets/"]
+    "exclude": ["node_modules/", "vendor/", "dist/", "build/", "test/assets/"],
+    "rules": {
+      "exclude": ["no-explicit-any", "no-unused-vars"]
```

Comment:
This was added to globally disable problematic rules that were generating hundreds of false positives in legacy code.